internal/graphicscommand/image.go

```diff
@@ -190,28 +190,25 @@ func (i *Image) IsInvalidated() bool {
 	return i.image.IsInvalidated()
 }
 
-// Dump dumps the image to the specified path.
-// In the path, '*' is replaced with the image's ID.
+// FormatPath replaces * in the given path with the Image's ID.
+func (i *Image) FormatPath(path string) string {
+	return strings.ReplaceAll(path, "*", strconv.Itoa(i.id))
+}
+
+// DumpBytes dumps the image as PNG data.
 //
 // If blackbg is true, any alpha values in the dumped image will be 255.
 //
 // This is for testing usage.
-func (i *Image) Dump(graphicsDriver graphicsdriver.Graphics, path string, blackbg bool, rect image.Rectangle) error {
+func (i *Image) DumpBytes(graphicsDriver graphicsdriver.Graphics, blackbg bool, rect image.Rectangle) ([]byte, error) {
```

Comment:
You don't have to change this function. Instead. Create `Dump` implementation of JS version.